addons/maptools/functions/fnc_wipeMarkers.sqf

```diff
@@ -0,0 +1,22 @@
+#include "../script_component.hpp"
+/*
+ * Author: LorenLuke
+ * Delete all markers on the Plotting Board.
+ *
+ * Arguments:
+ * None
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [CONTROL] call ACE_maptools_fnc_wipeMarkers
```

Comment:
```suggestion
 * [] call ACE_maptools_fnc_wipeMarkers
```

Reply:
I was going to save this for the second PR I am going to make, but I guess I'll clean it up in this one.